desktop/libs/metadata/src/metadata/optimizer_api.py

```diff
@@ -397,7 +399,7 @@ def _convert_queries(queries_data):
         if isinstance(guid, str):
           guid = guid.encode('utf-8')
         # unpack_guid uses '%016x:%016x' while optmizer api uses '%s:%s'.
-        original_query_id = '%s:%s' % struct.unpack(b"QQ", base64.decodestring(guid))
+        original_query_id = '%s:%s' % struct.unpack(b"QQ", decodebytes(guid))
```

Comment:
I have checked this one, decodestring and decodebytes both are giving same output in python3.8.

Reply:
Thanks